### PR TITLE
Selfhost onboarding UX/UI updates

### PR DESCRIPTION
**Pull request by deanhannigan**
## Description

Updating existing onboarding  self host UI/UX to utilise the new form and layout components design.

Screens updated: 

- Login
- Request password reset
- Reset Password
- Accept User Invite
  - We now allow users to set their name immediately and upon completing their acceptance, they will now be taken directly into the application
- Create admin

Addresses: 
- https://github.com/Budibase/budibase/issues/9228

## Screenshots

Login with multiple SSO configured

Request Password reset

Reset password

Accept User Invite

Accept User Invite - acceptance to login flow

Create Admin - the screen you see when provisioning a new budibase install.

### CODE REVIEW

**Review by shogunpurple — APPROVED** [review]
LGTM!

File: packages/builder/src/pages/builder/auth/login.svelte [hunk 1]
```diff
@@ -57,75 +64,96 @@
...
+        <FancyInput
+          label="Your work email"
+          value={formData.username}
+          on:change={e => {
+            formData = {
+              ...formData,
+              username: e.detail,
+            }
+          }}
+          validate={() => {
```
Comment (aptkingston) [inline packages/builder/src/pages/builder/auth/login.svelte:94]: Same as before, all you need to do is just return error strings, then invoke form.validate() and use the result to know wether the form is valid. You don't need to keep track of any error variables or pass any `error` props into the fields unless they're custom errors (such as returned from an API or something).

This is my bad - I musn't have explained that clearly enough.

File: packages/builder/src/pages/builder/auth/reset.svelte [hunk 2]
```diff
@@ -37,47 +49,92 @@
...
+          value={formData.password}
+          type="password"
+          on:change={e => {
+            formData = {
+              ...formData,
+              password: e.detail,
+            }
+          }}
+          validate={() => {
+            let fieldError = {}
```
Comment (aptkingston) [inline packages/builder/src/pages/builder/auth/reset.svelte:79]: Confirmation passwords are a bit of a different case so it's probably fine to use custom errors here, but again you probably don't need to.

**Review by aptkingston — APPROVED** [review]
The form validation could be done in a much neater way using the built in validation, but  that's my bad for obviously not explaining it properly! I can run you through that and we can refactor this to use it properly at a later stage, but I'm happy to get this merged now. No need to make it a blocker if this implementation also works.

I started writing comments for each usage but then saw that all the validation is done that same way so I stopped - but like I said we can jump on a call and refactor it all some other time.

Let's get this merged!

**Pull request merged**